packages/google_maps_flutter/google_maps_flutter_web/example/integration_test/google_maps_controller_test.mocks.dart

```diff
@@ -1,4 +1,4 @@
-// Mocks generated by Mockito 5.0.16 from annotations
+// Mocks generated by Mockito 5.1.0 from annotations
```

Comment:
These mocks are now generated with 5.2.0 IIRC, so this will conflict, but you can safely ignore them and re-generate.

Reply:
Regenerated